nym-vpn-desktop/src-tauri/src/commands/node_location.rs

```diff
@@ -68,7 +63,13 @@ pub async fn set_node_location(
 #[tauri::command]
 pub async fn get_fastest_node_location() -> Result<Country, CmdError> {
     debug!("get_fastest_node_location");
-    Ok(FASTEST_NODE_LOCATION.clone())
+    get_low_latency_entry_country().await.map_err(|e| {
```

Comment:
So I wonder about this, I hope this is not used in the UI to select the fastest country, but only for display purposes

Reply:
Fastest feature is still disabled in frontend, so it's not used yet